src/xGadgetron/cGadgetron/include/sirf/Gadgetron/encoding.h

```diff
@@ -0,0 +1,153 @@
+/*
+SyneRBI Synergistic Image Reconstruction Framework (SIRF)
+Copyright 2015 - 2021 Rutherford Appleton Laboratory STFC
+
+This is software developed for the Collaborative Computational
+Project in Synergistic Reconstruction for Biomedical Imaging (formerly CCP PETMR)
+(http://www.ccpsynerbi.ac.uk/).
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+*/
+
+/*!
+\file
+\ingroup Gadgetron Extensions
```

Comment:
probably no spaces allowed in group names

Reply:
i just copied from gadgetron_x.h where they were present.

Reply:
ok. I've created https://github.com/SyneRBI/SIRF/issues/886.
Let's not change this in this PR therefore